core/src/test/java/bisq/core/account/sign/SignedWitnessTest.java

```diff
@@ -10,22 +10,51 @@
 
 import java.time.Instant;
 
+import org.junit.Before;
 import org.junit.Test;
 
+import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 public class SignedWitnessTest {
-    @Test
-    public void testProtoRoundTrip() {
-        ECKey arbitrator1Key = new ECKey();
 
-        byte[] witnessOwner1PubKey = Sig.getPublicKeyBytes(Sig.generateKeyPair().getPublic());
+    private ECKey arbitrator1Key;
+    private byte[] witnessOwner1PubKey;
+    private byte[] witnessHash;
+    private byte[] witnessHashSignature;
+
+    @Before
+    public void setUp() {
+        arbitrator1Key = new ECKey();
+        witnessOwner1PubKey = Sig.getPublicKeyBytes(Sig.generateKeyPair().getPublic());
+        witnessHash = Utils.sha256hash160(new byte[]{1});
+        witnessHashSignature = arbitrator1Key.signMessage(Utilities.encodeToHex(witnessHash)).getBytes(Charsets.UTF_8);
+    }
 
-        byte[] witnessHash = Utils.sha256hash160(new byte[]{1});
-        byte[] witnessHashSignature = arbitrator1Key.signMessage(Utilities.encodeToHex(witnessHash)).getBytes(Charsets.UTF_8);
+    @Test
+    public void testProtoRoundTrip() {
         SignedWitness signedWitness = new SignedWitness(true, witnessHash, witnessHashSignature, arbitrator1Key.getPubKey(), witnessOwner1PubKey, Instant.now().getEpochSecond(), 100);
         assertEquals(signedWitness, SignedWitness.fromProto(signedWitness.toProtoMessage().getSignedWitness()));
+    }
 
+    @Test
+    public void isImmutable() {
+        byte[] signerPubkey = arbitrator1Key.getPubKey();
+        SignedWitness signedWitness = new SignedWitness(true, witnessHash, witnessHashSignature, signerPubkey, witnessOwner1PubKey, Instant.now().getEpochSecond(), 100);
+        byte[] witnessHash = signedWitness.getWitnessHash().clone();
```

Comment:
Maybe better to use a different name? Now the this.witnessHash and witnessHash are different, which makes the test a bit harder to read IMO.

Reply:
done